os-eap-s2i/added/s2i/assemble

```diff
@@ -21,6 +21,31 @@ if [ -f $LOCAL_SOURCE_DIR/configuration/settings.xml ]; then
   mv $LOCAL_SOURCE_DIR/configuration/settings.xml $HOME/.m2
 fi
 
+function addMavenServers() {
+  IFS=',' read -a repo_ids <<< $MAVEN_REPO_IDS
+  IFS=',' read -a repo_usernames <<< $MAVEN_REPO_USERNAMES
+  IFS=',' read -a repo_passwords <<< $MAVEN_REPO_PASSWORDS
+
+  if [ "${#repo_ids[@]}" -ne "0" ]; then
+    local count=0
+    for repo_id in ${repo_ids[@]}; do
+      if [ -n "${repo_usernames[$count]}" ] && [ -n "${repo_passwords[$count]}" ]; then
+        local xml="\n\
+          <server>\n\
+            <id>${repo_id}</id>\n\
+            <username>${repo_usernames[$count]}</username>\n\
+            <password>${repo_passwords[$count]}</password>\n\
+          </server>\n\
```

Comment:
This would result in the usernames/password being stored in the image which is likely not what is expected, it may be better just to refer to the environment variables within the settings.xml